docs/index.rst

```diff
@@ -25,30 +25,36 @@ To install Flocker, first choose which Cluster Manager you are using, or intend
 	    <div class="pod-boxout pod-boxout--orchestration pod-boxout--recommended">
 			<img src="_static/images/docker2x.png" alt="Docker logo"/>
 			<span>Docker Swarm, with Docker Compose<em>Try it Now</em></span>
```

Comment:
In the mock-up this text was also changed, I can't tell if that change was significant or not though. (Additional links for installing flocker, and a bulleted list of things covered in the tutorial.

Feel free to leave as is if this is intentional, just wanted to point it out in case you missed it.